string msg_string
int32 service_id
---
bool status
